src/main/java/mate/academy/springintro/SpringIntroApplication.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.springintro;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SpringIntroApplication {
+    public static void main(String[] args) {
+        SpringApplication.run(SpringIntroApplication.class, args);
+    }
```

Comment:
You should implement task description
`Class with @SpringBootApplication annotation that will have a CommandLineRunner bean.`

Reply:
not fixed